packages/action/src/index.ts

```diff
@@ -23,6 +23,7 @@ export function* run(): Generator<any, any, any> {
         ? process.env.GITHUB_TOKEN || ""
         : core.getInput("token");
     const inputCommand = core.getInput("command");
+    const releaseCommitish = core.getInput('releaseCommitish');
```

Comment:
I think maybe we move the fallback here, as I suspect we will end up using this more than just in the release creation. Ideally we move the pull_request creation into the covector action, and have some consideration of this.